### DISCUSSION TITLE
Convenience function to read ANSYS files

### DISCUSSION THREAD

**Discussion by johannes-mueller** [boschresearch/pylife#57] [Ideas]
In [demos/import_mesh_ansys.ipynb](https://github.com/boschresearch/pylife/tree/develop/demos/import_mesh_ansys.ipynb) we dedicate almost 20 lines of code to convert the ANSYS mesh data structure to a pyLife compliant mesh dataframe. 

Maybe we should have some kind of convenience function for it.

**Comment by DKreuter** [comment]
Good idea, we are using[PyMAPDL Reader - Legacy Binary and Archive File Reader](https://reader.docs.pyansys.com/version/stable/), actually. We should shift to the actual version of [pymapdl](https://mapdl.docs.pyansys.com/version/stable/), but we should double check, if we can read *rst files w/o Ansys license.

**Reply by MiguelDLM** [reply]
Is this possible? I'm exploring the data of the example provided (beam_3d.rst) to better understand the data structure I need to apply to my results in MSH format.

**Reply by DKreuter** [reply]
Yes it should, just try it out 😉

**Reply by MiguelDLM** [reply]
Oh it works! I didn't try because of the warning in the instructions (`#%% Ansys (license is necessary)`)

Thanks!
